src/Compilers/CSharp/Portable/Binder/Binder_TupleOperators.cs

```diff
@@ -196,6 +249,29 @@ internal partial class Binder
             return new TupleBinaryOperatorInfo.Multiple(operators, leftTupleType, rightTupleType);
         }
 
+        internal static BoundExpression GiveTupleTypeToDefaultLiteralIfNeeded(BoundExpression expr, TypeSymbol targetType)
```

Comment:
>internal [](start = 8, length = 8)

Do you expect this method to be used outside of  the Binder type?

Reply:
Found it

---
In reply to: [173292507](https://github.com/dotnet/roslyn/pull/25327#discussion_r173292507) [](ancestors = 173292507)